packages/core/src/reducers/i18n.ts

```diff
@@ -0,0 +1,24 @@
+import { SET_LOCALE, SET_TRANSLATIONS } from '../actions';
+
+export const i18nReducer = (
+  state = {},
```

Comment:
I know I said otherwise the last time but I think the default intialization of the `locale` should happen here. Reason being is that users probably will not always use the `initJsonFormsStore` function to initialize the store and hence no default locale will be set.